src/extension.ts

```diff
@@ -79,6 +80,13 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
         context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider('java', javaDebugProvider));
         context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider('PowerShell', powershellDebugProvider));
         context.subscriptions.push(vscode.workspace.registerTaskProvider(func, new FuncTaskProvider(nodeDebugProvider, pythonDebugProvider, javaDebugProvider, powershellDebugProvider)));
+
+        // Temporary workaround so that "powerShellVersion" is an allowed property on SiteConfig
+        // https://github.com/Azure/azure-sdk-for-js/issues/10552
+        // tslint:disable-next-line: no-non-null-assertion
+        WebSiteManagementMappers.SiteConfig.type.modelProperties!.powerShellVersion = { serializedName: 'powerShellVersion', type: { name: 'String' } };
```

Comment:
I was trying to find documentation for `WebSiteManagementMappers` to understand it but was struggling to find some. So adding `powerShellVersion` here makes that property available without throwing build errors?

Reply:
At the end of the day it's just a temporary workaround that should go away once they release the next version of "arm-appservice". But I'll try to explain how mappers work with a mock example (I didn't find docs - I just went digging on the sdk). Let's say this is the mapper:
```
export const AddressMapper: msRest.CompositeMapper = {
  serializedName: "Address",
  type: {
    name: "Composite",
    className: "Addresss",
    modelProperties: {
      street: {
        serializedName: "street",
        type: {
          name: "String"
        }
      },
      zipCode: {
        serializedName: "zipCode",
        type: {
          name: "String"
        }
      }
    }
};
```

And I make an api call like this:
```
const address: Address = { street: "105th Ave", zipCode: "55555", badProperty: "oops" };
await client.addAddressToContacts(address);
```

The sdk uses the mapper inside "addAddressToContacts" to convert the object into a string request body. The mapper is essentially a schema used to do some extra validation and cleanup (aka removing "badProperty") before doing a JSON.stringify on the object and sending the request.

My problem is that the sdk thinks "powerShellVersion" is a bad property, so it removes it from the object (even though I _did_ set it) before sending the request. This has nothing to do with build errors. It has to do with the actual functionality of sending the request.